packages/playwright-test/src/reporters/html.ts

```diff
@@ -228,7 +229,7 @@ class HtmlBuilder {
     this._dataZipFile = new yazl.ZipFile();
   }
 
-  async build(rawReports: JsonReport[]): Promise<{ ok: boolean, singleTestId: string | undefined }> {
+  async build(runLevelReport: { attachments: JsonAttachment[] }, rawReports: JsonReport[]): Promise<{ ok: boolean, singleTestId: string | undefined }> {
```

Comment:
curious: why isn't the first argument just a `testReportAttachments: JsonAttachment[]`?

Reply:
I named it poorly! Updated! Thanks!